src/mlpack/methods/ann/init_rules/gaussian_init.hpp

```diff
@@ -76,10 +76,10 @@ class GaussianInitialization
 
  private:
   //! Mean of the gaussian.
-  const double mean;
+  double mean;
```

Comment:
class with const member variable doesn't support assignment operator, so I just removed the const qualifier.